src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/DataAppSI.cs

```diff
@@ -364,5 +366,38 @@ public async Task<DataElement> Update(Instance instance, DataElement dataElement
 
             throw new PlatformHttpException(response);
         }
+
+        private static string GetComplientContentHeader(string headerValues)
+        {
+            // Remove all spaces from filename
+            StringBuilder bld = new StringBuilder();
+            string keyWord = "filename=";
+            int splitIndex = headerValues.IndexOf("filename=") + keyWord.Length;
```

Comment:
Use the variable keyWord instead of the constant

Reply:
thanks. found a few other bugs.. Looking into writing some tests so you can hang back for a minute with the review

Reply:
And we're back on track. Moved the logic tho :)